src/components/pages/Projects.css

```diff
@@ -0,0 +1,30 @@
+h5 {
+    text-align: left;
+    font-family: Bahnschrift;
+    font-size: 15px;
+    font-weight: bold;
+    padding: 0px 15px 15px;
+    margin-left: 90px;
+    margin-right: 90px;
+    display: inline-block;
+  }
+
+.info_wrapper > p {
```

Comment:
One worrying thing that I noticed and didn't know about before: these class attributes seem to be affected by other CSS classes with the same name from other CSS files (eg. JoinUs.css). From now on, let's make sure that each class has a distinct name.